symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -526,6 +531,118 @@ public void updateStatusOfUser(@Nonnull Long uid, @Nonnull UserStatus status) {
         () -> userApi.v1AdminUserUidStatusUpdatePost(authSession.getSessionToken(), uid, status));
   }
 
+  /**
+   * Returns the list of followers of a specific user.
+   *
+   * @param uid   User Id
+   * @return The list of followers of a specific user with the pagination information.
+   * @see <a href="https://developers.symphony.com/restapi/v20.9/reference#list-user-followers">List User Followers</a>
+   */
+  public FollowersListResponse listUserFollowers(@Nonnull Long uid) {
```

Comment:
if we don't pass any value for pagination we return all the followers?

Reply:
It isn't documented on developers site, but I guess so.

Reply:
About the `limit`:

> If no “limit“ parameter specified in the path of the request, it will be set by default to 100